website/src/components/EventSlider/index.tsx

```diff
@@ -0,0 +1,125 @@
+import React from "react";
+import Slider from "react-slick";
+import { useTranslation } from "react-i18next";
+import { Typography, Link, Box } from "@material-ui/core";
+import useStyles from "./style";
+import events from '../../resources/events.json';
+
+const EventSlider: React.FC = () => {
+  const { t } = useTranslation();
+  const classes = useStyles();
+
+  const NextArrow = (props: any) => {
+    const { className, style, onClick } = props;
+    return (
+      <div
+        className={className}
+        style={{ ...style, display: "block" }}
+        onClick={onClick}
+      >
+        <img
+          src="../Images/svg/right_arrow.svg"
+          alt={t("home.adaptorsTestimonials.nextArrowAlt")}
+        />
+      </div>
+    );
+  };
+
+  const PrevArrow = (props: any) => {
+    const { className, style, onClick } = props;
+    return (
+      <div
+        className={className}
+        style={{ ...style, display: "block" }}
+        onClick={onClick}
+      >
+        <img
+          src="../Images/svg/left_arrow.svg"
+          alt={t("home.adaptorsTestimonials.previousArrowAlt")}
+        />
+      </div>
+    );
+  };
+
+  const settings = {
+    infinite: true,
+    autoplay: false,
```

Comment:
What is the implication of this autoplay ?

Reply:
I thought for events, the automatic sliding option won't be needed.